dspace/modules/xmlui/src/main/webapp/themes/Mirage/Mirage.xsl

```diff
@@ -576,7 +548,7 @@
     <xsl:template match="dri:options/dri:list[@n='DryadConnect']" priority="3">
       <div class="NOT-simple-box">
         <!-- START CONNECT  -->
-        <h1 class="ds-div-head ds_connect_with_dryad_head" id="ds_connect_with_dryad_head">Be part of Dryad
+        <h1 class="ds-div-head ds_connect_with_dryad_head" id="ds_connect_with_dryad_head">Tweets from Dryad
```

Comment:
I think that this one is causing the sidebar to say "Tweets from Dryad" even on pages where the sidebar doesn't have tweets, like membershipOverview.